source_test.yaml

```diff
@@ -181,7 +193,7 @@
 - name: 'malicious-packages'
   versions_from_repo: False
   type: 0
-  ignore_patterns: ['^(?!MAL-).*$', 'MAL-0000.*']
+  ignore_patterns: ['^(?!MAL-).*$']
```

Comment:
should we bring back the MAL-0000 ? I think that's still needed.

Reply:
From memory, we found that having multiple items in this list did not achieve the desired effect. I now note that the `python` source is still in this state (in the reverse order) so I'll need to check how that's working out... I think the regex needs to be merged into a single one (in both cases).

Reply:
Hmm, I think it should work as intended in this case, because we ignore the file if **any** of the patterns in the list match. 

MAL-0000- (and PYSEC-0000-) entries will exist and represent intermediate advisories that don't yet have a property ID assigned, so we need to ignore them.

Reply:
ref: https://github.com/google/osv.dev/blob/34bd002b44baf7f0752614c43a1151f6f2aeca2d/osv/models.py#L855

Reply:
Ack. a5419a21 adjusts accordingly.